playground/src/Playground.tsx

```diff
@@ -1,9 +1,10 @@
-import React, { CSSProperties } from 'react';
+import React, { CSSProperties, useEffect, useRef } from 'react';
+import ReactDOM from 'react-dom';
 import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import oceanicNext from 'prism-react-renderer/themes/oceanicNext';
-import './playground.css';
+import './dockit-react-playground';
 
-const styles = {
+const playgroundStyles = {
```

Comment:
I had to rename it, because where it's used I already have a shadowed var `styles` from props